docs/api/c/struct_ort_api.html

```diff
@@ -8123,7 +8123,6 @@ <h2 class="memtitle"><span class="permalink"><a href="#a5f357353ba83dce8103d0faf
 </dl>
 <p>Currently supported providers: SNPE XNNPACK</p>
 <p >Note: If an execution provider has a dedicated SessionOptionsAppendExecutionProvider_&lt;provider name&gt; function that should be used to add it.</p>
-<p >SNPE supported keys: "runtime": SNPE runtime engine, options: "CPU", "CPU_FLOAT32", "GPU", "GPU_FLOAT32_16_HYBRID", "GPU_FLOAT16", "DSP", "DSP_FIXED8_TF", "AIP_FIXED_TF", "AIP_FIXED8_TF". Mapping to SNPE Runtime_t definition: CPU, CPU_FLOAT32 =&gt; zdl::DlSystem::Runtime_t::CPU; GPU, GPU_FLOAT32_16_HYBRID =&gt; zdl::DlSystem::Runtime_t::GPU; GPU_FLOAT16 =&gt; zdl::DlSystem::Runtime_t::GPU_FLOAT16; DSP, DSP_FIXED8_TF =&gt; zdl::DlSystem::Runtime_t::DSP. AIP_FIXED_TF, AIP_FIXED8_TF =&gt; zdl::DlSystem::Runtime_t::AIP_FIXED_TF. SNPE Runtime_t refers to <a href="https://developer.qualcomm.com/docs/snpe/group__c__plus__plus__apis.html">https://developer.qualcomm.com/docs/snpe/group__c__plus__plus__apis.html</a> "priority": execution priority, options: "low", "normal". "buffer_type": ITensor or user buffers, options: "ITENSOR", user buffer with different types - "TF8", "TF16", "UINT8", "FLOAT". "ITENSOR" &ndash; default, ITensor which is float only. "TF8" &ndash; quantized model required, "FLOAT" &ndash; for both quantized or non-quantized model If SNPE is not available (due to a non Snpe enabled build or its dependencies not being installed), this function will fail.</p>
```

Comment:
we only need to remove: SNPE Runtime_t refers to <a href="https://developer.qualcomm.com/docs/snpe/group__c__plus__plus__apis.html">https://developer.qualcomm.com/docs/snpe/group__c__plus__plus__apis.html</a>